frontend/store/vaults.ts

```diff
@@ -50,13 +59,21 @@ export const actions = {
             commit('setIsVaultLoading', false);
         }
     },
-    async liquidateVault({ rootGetters, commit, getters, dispatch }: ActionContext<State, State>, vaultId: number) {
+    async liquidateVault(
+        { rootGetters, commit, getters, dispatch }: ActionContext<State, State>,
+        { vaultId, walletAddress }: { vaultId: number; walletAddress?: string }
+    ) {
         const network = rootGetters['network/getMakerNetwork'];
-        const walletAddress = rootGetters['wallet/getAddress'];
+        const executeWalletAddress = walletAddress || rootGetters['wallet/getAddress'];
```

Comment:
Actually, you don't need to get wallet address from the store, `rootGetters['wallet/getAddress']` because `liquidateVault` core function doesn't require `walletAddress` argument and will substitute with the signer vault address (which is better in case we have store-related problems)